dgl_sparse/src/elemenwise_op.cc

```diff
@@ -24,7 +24,10 @@ c10::intrusive_ptr<SparseMatrix> SpSpAdd(
   ElementwiseOpSanityCheck(A, B);
   auto torch_A = COOToTorchCOO(A->COOPtr(), A->value());
   auto torch_B = COOToTorchCOO(B->COOPtr(), B->value());
-  auto sum = (torch_A + torch_B).coalesce();
+  auto sum = torch_A + torch_B;
+  torch_A = torch::Tensor();
```

Comment:
This is a little bit ugly.

Reply:
@Rhett-Ying

Reply:
how about wrap `torch_A/B` into a `{}` then they'll be destructed automatically once leaves the domain.

Reply:
`lambda` could be another alternative.